index.js

```diff
@@ -102,8 +102,12 @@ function send(ctx, path, opts) {
  * Check if it's hidden.
  */
 
-function leadingDot(path) {
-  return '.' == basename(path)[0];
+function isHidden(path) {
+  // unescaped version: /(^|[/\]).(?!.[/\])/
+  // (^|[\/]) matches a path separator or start of the string, . matches leading dot
+  // while (?!.[/\]) makes sure that something like /../ should not be matched
+  // and is passed to resolve-path to get the correct error response
+  return /(^|[\\\/])\.(?!\.[\\\/])/.test(path);
 }
```

Comment:
@andyhu you check path before `normalize`, so `/./` will treated as `is hidden`.

I'd rather this
https://github.com/pillarjs/send/blob/master/index.js#L736-L744

wait @jonathanong 's decision